topology/pynest/hl_api.py

```diff
@@ -1744,6 +1744,18 @@ def _draw_extent(ax, xctr, yctr, xext, yext):
            xticks=tuple(), yticks=tuple())
 
 
+def _get_newpos(pos, ext):
```

Comment:
I would call it `_shifted_positions()`, that also makes clear it returns multiple positions.